library/swift/src/filters/ResponseFilter.swift

```diff
@@ -1,7 +1,28 @@
+@_implementationOnly import EnvoyEngine
 import Foundation
 
+extension EnvoyHTTPFilter {
+  /// Initializer
+  convenience init(responseFilter: ResponseFilter) {
+    self.init()
+    self.name = responseFilter.name
+    self.onResponseHeaders = {
+      let result = responseFilter.onResponseHeaders(ResponseHeaders(headers: $0), endStream: $1)
```

Comment:
nit: let's name these parameters in the opening of the closure

Reply:
sgtm